### ISSUE TITLE
Exceptions should inherit from StandardError

### ISSUE THREAD

**Issue by djohnson** [daisybill/daisybill_api#60]
The exceptions raised by this gem, such as InternalServerError, all inherit directly from Exception rather then StandardError.  This causes some unexpected behavior in regards to rescuing as it requires these classes to be explicitly named in the rescue.  

Typical rescue block
```ruby
begin
  #some daisybill method that raises
  DaisybillApi::Models::ClaimsAdministrator.all
rescue
  # will never be rescued since the daisybill errors aren't StandardErrors
end
```

What is actually required right now
```ruby
begin
  #some daisybill method that raises
  DaisybillApi::Models::ClaimsAdministrator.all
rescue StandardError, DaisybillApi::Data::Client::BasicError
  # will be rescued due to explicit Exception naming
end
```

**Comment by coreyleveen** [comment]
@djohnson Thanks for reporting these issues. I'll look into it and will get back to you when I have an update

**Comment by coreyleveen** [comment]
@djohnson Fixed in #62
